### PR TITLE
PAYEE-365 add govukDateInput component , gds styles,  fix tests

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/employments/EndEmploymentView.scala.html
Comment (brendanstraughan) [inline app/views/employments/EndEmploymentView.scala.html:30]: was the header meant to be removed?

File: app/views/employments/EndEmploymentView.scala.html
Comment (brendanstraughan) [inline app/views/employments/EndEmploymentView.scala.html:47]: I think this will fail the build as backLinkContent is now a required field

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala
Comment (masahmad) [reply test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala:77]: Ooops - checking now

File: app/views/employments/EndEmploymentView.scala.html
Comment (masahmad) [reply app/views/employments/EndEmploymentView.scala.html:30]: yes its not used in pages where we are using a form as the H1 heading is handled by the Form field legend - this is to prevent duplication of  the heading and was originally an issue reported from the accessibility team. Also as we're unlikely to have the print button on the form pages  - there is no need for the header anymore

File: app/views/employments/EndEmploymentView.scala.html
Comment (brendanstraughan) [inline app/views/employments/EndEmploymentView.scala.html:48]: needs to be a Some() I believe

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala
Comment (masahmad) [reply test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala]: I've uncommented the test  and pushed however (it fails)  we have to wait until ticket PAYEE-359 is merged as that has an update to the same test that will solve the issue.

https://github.com/hmrc/tai-frontend/pull/898/files#diff-b6b05415cca2409ac354167559257c7557055052ec49b74dac83b6d77c993340L63

File: app/views/employments/EndEmploymentView.scala.html
Comment (masahmad) [reply app/views/employments/EndEmploymentView.scala.html]: thanks added

File: app/views/employments/EndEmploymentView.scala.html
Comment (masahmad) [reply app/views/employments/EndEmploymentView.scala.html:47]: added

File: app/views/employments/EndEmploymentView.scala.html
Comment (sergi-hmrc) [inline app/views/employments/EndEmploymentView.scala.html:37]: is this div needed? i think the Hint puts everything inside one (i might be wrong)

File: app/views/employments/EndEmploymentView.scala.html
Comment (sergi-hmrc) [inline app/views/employments/EndEmploymentView.scala.html:70]: are classes needed? i think if you use errorMessage with a filter by key should highlight only the one that has an error

Comment (masahmad) [reply app/views/employments/EndEmploymentView.scala.html:70]: discussed with Sergi - we can ignore this as it may not be possible with this component.

Comment (sergi-hmrc) [reply app/views/employments/EndEmploymentView.scala.html:70]: yup here's an example: https://github.com/hmrc/manage-transit-movements-amendment-frontend/blob/19e8c66587ba6647cd763ecc9a61a90a6922083c/app/viewmodels/govuk/DateFluency.scala

InputItem doesn't allow an errorMessage so LGTM

**Review by sergi-hmrc — APPROVED** [review]

**Review by brendanstraughan — APPROVED** [review]

**Pull request merged**
